targets/frontend/src/pages/unthemed.js

```diff
@@ -0,0 +1,142 @@
+import { getLabelBySource, SOURCES } from "@socialgouv/cdtn-sources";
+import { useForm } from "react-hook-form";
+import { Button } from "src/components/button";
+import { ThemePicker } from "src/components/forms/ContentPicker/ThemePicker";
+import { getUnthemedContentQuery } from "src/components/home/UnThemedContent";
+import { Layout } from "src/components/layout/auth.layout";
+import { Stack } from "src/components/layout/Stack";
+import { Li, List } from "src/components/list";
+import { withCustomUrqlClient } from "src/hoc/CustomUrqlClient";
+import { withUserProvider } from "src/hoc/UserProvider";
+import { RELATIONS } from "src/lib/relations";
+import { Box, Flex, Heading, Message, Spinner, Text } from "theme-ui";
+import { useMutation, useQuery } from "urql";
+
+const insertRelationMutation = `
+mutation insertRelation ($relations: [document_relations_insert_input!]!){
+  insert_document_relations(objects: $relations) {
+    affected_rows
+  }
+}
+`;
+
+export function UnthemedPage() {
+  const { handleSubmit, control } = useForm();
+
+  const [result, reexecuteQuery] = useQuery({
+    query: getUnthemedContentQuery,
+    variables: {
+      themeSources: [
+        SOURCES.SHEET_MT_PAGE,
+        SOURCES.SHEET_SP,
+        SOURCES.CONTRIBUTIONS,
+        SOURCES.EDITORIAL_CONTENT,
+        SOURCES.LETTERS,
+        SOURCES.EXTERNALS,
+        SOURCES.TOOLS,
+      ],
+    },
+  });
+
+  const [resultInsert, insertRelations] = useMutation(insertRelationMutation);
+
+  function onSubmit(data) {
+    const themedDocuments = Object.entries(data).flatMap(
+      ([contentId, theme]) => {
+        if (!theme) return [];
+        return {
+          data: { position: theme.position },
+          document_a: theme.cdtnId,
+          document_b: contentId,
+          type: RELATIONS.THEME_CONTENT,
+        };
+      }
+    );
+    insertRelations({ relations: themedDocuments }).then(() => {
+      console.log("fetch");
+      reexecuteQuery({ requestPolicy: "network-only" });
+    });
+  }
+  const { data, fetching, error } = result;
+  const documentMap =
+    data?.documents.reduce((state, { cdtnId, source, title }) => {
+      if (Object.prototype.hasOwnProperty.call(state, source)) {
```

Comment:
Perso je préfère:
```suggestion
      if (Object.keys(state).source) {
```
`Object.keys` ne remonte pas la chaine prototypale

Reply:
mais du coup il faudrait plutot faire `Object.keys(state).includes(source)` ? Normalement j'utilise `state.hasOwnProperty(source)` mais il me semble que ca fait râler eslint. 
autre option on vire cette regle :D

Reply:
oui pardon ! c'est juste pour la lisibilité, les `.call` ça me demande toujours un petit effort mental en plus. Il y a une raison pour laquelle on a interdit le .hasOwnProperty ?

Reply:
https://eslint.org/docs/rules/no-prototype-builtins en soit la raison me semble plutôt bonne. Sait-on jamais, comme tu le sens mais perso je préfère `Object.keys(state).includes("source")`

Reply:
Sinon autre option peut-être un peu plus risqué mais simplissime `if(state.source){`